src/zenml/zen_stores/rest_zen_store.py

```diff
@@ -404,7 +405,7 @@ def create_stack(
             request_model=CreateStackRequest,
         )
 
-    def get_stack(self, stack_id: UUID) -> StackModel:
+    def get_stack(self, stack_id: UUID, hydrated: bool = False) -> StackModel:
```

Comment:
Does `get_stack` also support the `hydrated` flag? If yes we should probably also include it on the `SqlZenStore`, correct?

Reply:
I agree, this is probably a left-over.